api4/channel_test.go

```diff
@@ -925,6 +925,38 @@ func TestGetAllChannels(t *testing.T) {
 		channels, resp = client.GetAllChannels(10000, 10000, "")
 		CheckNoError(t, resp)
 		require.Empty(t, *channels)
+
+		channels, resp = client.GetAllChannels(0, 10000, "")
+		require.Nil(t, resp.Error)
+		beforeCount := len(*channels)
+
+		var firstChannel model.Channel
+		for _, item := range *channels {
+			firstChannel = item.Channel
+			break
+		}
```

Comment:
why not just do `firstChannel := *channels[0].Channel`

Reply:
Meant to switch that, thanks.